### PR TITLE
fix(#6) - Changes for generator README.md and other fixes

### LINKED ISSUE #6: Changes for generator README.md and other fixes
Changes to the README required

## Additional context

Screenshot 2020-07-09 at 7 17 58 PM

### PR DESCRIPTION
**Pull request by hybridx**
## Closes #6 

 Added fix for .gitignore file and fixed README

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Ready-for-merge Checklist
+
+- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
+- [ ] Does the change have appropriate unit tests?
+- [ ] Did tests pass?
+- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
+- [ ] My changes generate no new warnings
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (gisumit) [inline .github/PULL_REQUEST_TEMPLATE.md:41]: I have **documented** my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.

Comment (hybridx) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: @gisumit I have updated the code with
```
- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
```

**Comment by hybridx** [comment]
@gisumit the PR is updated and waiting on your review

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Ready-for-merge Checklist
+
+- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
+- [ ] Does the change have appropriate unit tests?
+- [ ] Did tests pass?
+- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
+- [ ] My changes generate no new warnings
+- [ ] I have performed a self-review of my own code
+- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
```
Comment (diwanshi) [inline .github/PULL_REQUEST_TEMPLATE.md:41]: @hybridx can you add a checklist item for Web Accessibility?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+#### If it is a new component
+- [ ] Does your component follow One Platform style guidelines?
+- [ ] Is your component responsive (supports standard - Mobile, Tab, Desktop)
```
Comment (diwanshi) [inline .github/PULL_REQUEST_TEMPLATE.md:44]: @hybridx i don't think we'd be able to acheive mobile responsiveness. For now i think we can just go ahead with web browser support

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+#### Browsers you have tested in
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Edge
```
Comment (diwanshi) [inline .github/PULL_REQUEST_TEMPLATE.md:48]: there should be Safari as well as IE11

Comment (hybridx) [reply .github/PULL_REQUEST_TEMPLATE.md:48]: We can add Safari but I’m not sure we can test it in IE11

Screenshot 2020-07-20 at 11 37 02 AM

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (hybridx) [reply .github/PULL_REQUEST_TEMPLATE.md:44]: I don't like the idea of not supporting it from the start, but for now I have removed the responsiveness checklist item

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (hybridx) [reply .github/PULL_REQUEST_TEMPLATE.md:41]: @diwanshi  Added the following 
```
- [ ] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
+- [ ] My changes generate no new warnings
+- [ ] I have performed a self-review of my own code
+- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
+#### If it is a new component
+- [ ] Does your component follow One Platform style guidelines?
+- [ ] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
+- [ ] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
+#### Browsers you have tested in
+- [ ] Chrome [Latest 2 versions]
```
Comment (diwanshi) [inline .github/PULL_REQUEST_TEMPLATE.md:47]: There should be a checklist item for latest two versions supported by rhel distribution

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- [ ] Firefox [Latest 2 versions]
+- [ ] Edge [Latest 2 versions]
+- [ ] Safari [Latest 2 versions]
+- [ ] IE [Latest 2 versions]
```
Comment (diwanshi) [inline .github/PULL_REQUEST_TEMPLATE.md:51]: it should only support IE11 and not other versions

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (hybridx) [reply .github/PULL_REQUEST_TEMPLATE.md:47]: @diwanshi Added the browsers listed here: https://access.redhat.com/help/browsers

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (hybridx) [reply .github/PULL_REQUEST_TEMPLATE.md:51]: Changed it to 
```
- [ ] Google Chrome [Latest 2 versions]
- [ ] Mozilla Firefox [Latest 2 versions]
- [ ] Microsoft Edge [Latest 2 versions]
- [ ] Apple Safari [Latest 2 versions]
- [ ] Microsoft Internet Explorer 11
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
+- [ ] My changes generate no new warnings
+- [ ] I have performed a self-review of my own code
+- [ ] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
+#### If it is a new component
+- [ ] Does your component follow One Platform style guidelines?
+- [ ] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
+- [ ] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
+#### Browsers you have tested in
+- [ ] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
```
Comment (diwanshi) [inline .github/PULL_REQUEST_TEMPLATE.md:47]: u missed to add Firefox and Chrome's latest two versions

**Pull request merged**